packages/wdio-cli/src/commands/run.js

```diff
@@ -82,6 +82,10 @@ export const builder = {
     },
     cucumberOpts: {
         desc: 'Cucumber options'
+    },
+    yes: {
```

Comment:
given recent changes to the cli module I think this needs to go in the config command, no?

Reply:
Yes, this should be in the `config` command as this flag is only intended to be used there.

Reply:
Please also see #4587 as I am making some changes to the command builder. But I don't mind figuring out the conflicts

Reply:
@christian-bromann @baruchvlz implemented in 75566ea